environment-dev.yml

```diff
@@ -21,15 +21,15 @@ dependencies:
   - netCDF4
   - numcodecs
   - numpy
-  - pandas >=2.0,<2.2
+  - pandas >=2.2
   - parse
   - pyyaml
   - rechunker
   - scipy
   - shapely >=2.0
   - sparse
   - toolz
-  - xarray <2023.11.0  # FIXME: Remove when pandas 2.2 is released and xclim is fixed.
+  - xarray
```

Comment:
It might be worthwhile to set the lower bounds for this pin.

Reply:
What would be the right bound, 2023.11.0?